libs/wingsdk/src/target-tf-aws/table.ts

```diff
@@ -56,7 +56,10 @@ export class Table extends ex.Table {
       throw new Error("tables can only be bound by tfaws.Function for now");
     }
 
-    if (ops.includes(ex.TableInflightMethods.INSERT)) {
+    if (
+      ops.includes(ex.TableInflightMethods.INSERT) ||
+      ops.includes(ex.TableInflightMethods.UPSERT)
+    ) {
```

Comment:
Can we add unit tests for this?

Reply:
how? (since there is no automatic way as far as I know to know which permissions are needed for each sdk method?) The wing SDK test caught it, we just need to run them on time....